### DISCUSSION TITLE
How are you all getting MDM CSR approval?

### DISCUSSION THREAD

**Discussion by brettfire** [micromdm/micromdm#775] [Q&A] [answered]
Apple is saying that you need the enterprise developer account, which has a pre-req that you have 100+ employees. This is an absolutely ridiculous requirement in my opinion. I would assume that many of those working on this project aren't working in an org with 100+ employees, so can anyone enlighten me on the best way to get MDM CSR capabilities so I can implement this?!
Thanks!

**Comment by korylprince** [comment, marked as answer]
Recently on the macadmins Slack someone posted about this issue and @groob said "you have to counter and tell them you're only applying for an enterprise account because you need the MDM CSR". 

I can't speak to this as we have over 100 employees. They didn't ask too many questions after I told them we just needed the MDM CSR to manage our MacBooks.

**Comment by korylprince** [comment]
A recent development on this topic: Apple is now granting the MDM CSR for normal developer accounts. Currently, the recommended method to request the MDM CSR:

- Go to https://developer.apple.com/contact/
- Go to Membership and Account => Other Membership or account questions => Email
- Request the MDM CSR for your Developer Account/Team ID

**Reply by korylprince** [reply]
By "normal developer account", I mean Apple's standard Developer Account, that you would sign up for [here](https://developer.apple.com/programs/enroll/).

This is different from an Enterprise Developer Account, which can be harder to get and was previously needed to get an MDM CSR.

**Reply by GregOriol** [reply]
I can confirm that

**Reply by mnafees** [reply]
How long is the turn-around time, generally speaking?
